src/Akka.Logger.Serilog/SerilogLoggingAdapter.cs

```diff
@@ -18,86 +22,59 @@ private class ContextNode
             public ILogEventEnricher Enricher { get; set; }
         }
 
-        private readonly ILoggingAdapter adapter;
-        private readonly ContextNode enricherNode;
+        private readonly ContextNode _enricherNode;
 
-        public SerilogLoggingAdapter(ILoggingAdapter adapter)
-            : this(adapter, null)
-        {
-        }
 
-        private SerilogLoggingAdapter(ILoggingAdapter adapter, ContextNode enricherNode)
+        public SerilogLoggingAdapter(LoggingBus bus, string logSource, Type logClass) : this(bus, logSource, logClass, null)
         {
-            this.adapter = adapter;
-            this.enricherNode = enricherNode;
         }
 
-        /// <summary>
-        /// Check to determine whether the <see cref="F:Akka.Event.LogLevel.DebugLevel" /> is enabled.
-        /// </summary>
-        public bool IsDebugEnabled => adapter.IsDebugEnabled;
-
-        /// <summary>
-        /// Check to determine whether the <see cref="F:Akka.Event.LogLevel.InfoLevel" /> is enabled.
-        /// </summary>
-        public bool IsInfoEnabled => adapter.IsInfoEnabled;
-
-        /// <summary>
-        /// Check to determine whether the <see cref="F:Akka.Event.LogLevel.WarningLevel" /> is enabled.
-        /// </summary>
-        public bool IsWarningEnabled => adapter.IsWarningEnabled;
-
-        /// <summary>
-        /// Check to determine whether the <see cref="F:Akka.Event.LogLevel.ErrorLevel" /> is enabled.
-        /// </summary>
-        public bool IsErrorEnabled => adapter.IsErrorEnabled;
-
-        /// <summary>Determines whether a specific log level is enabled.</summary>
-        /// <param name="logLevel">The log level that is being checked.</param>
-        /// <returns><c>true</c> if the specified level is enabled; otherwise <c>false</c>.</returns>
-        public bool IsEnabled(LogLevel logLevel)
+        private SerilogLoggingAdapter(LoggingBus bus, string logSource, Type logClass, ContextNode enricher) : base(bus, logSource, logClass, SerilogLogMessageFormatter.Instance)
```

Comment:
We pass a `SerilogLogMessageFormatter` singleton instance down to the base class here, so there's no need to request or configure one from the outside. Handled internally.

Reply:
Thats a good addition. Makes the api simpler to use.